### ISSUE TITLE
No Internet Connection Bug

### ISSUE THREAD

**Issue by Ch4osCosmo** [mitchazj/truffle#21]
Truffle will recognise my internet connection when I open the app and then after some time it will no longer see my internet connection (even though there has been no change in my connection)

**Comment by mitchazj** [comment]
Yeah a lot of people have been reporting this. It's occurring because QUT recently started blocking ping requests to their domain, which is what Truffle was using (every 5 seconds or so) to check the internet connection. If you check out the dev branch you'll see I've written a new method that should hopefully work everywhere, just waiting on a few things before pushing out an update.
